.github/workflows/deployment-jdk-ea.yml

```diff
@@ -0,0 +1,328 @@
+# This workflow is a clone of "deployment.yml"
+# The difference is that this workflow uses JDK early access builds (jdk-ea) to check the build of JabRef
+# We separated this from the main workflow as we do not want to check on each PR if the JDK build, but only on main
+name: Deployment (JDK early access builds)
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
We should make this a once in a while workflow to not pollute our builds.jabref.org

Reply:
Changed to each Tuesday morning.